devtools_m1/cypress/integration/Wishlists/steps.js

```diff
@@ -0,0 +1,98 @@
+import { Given, When, Then } from "cypress-cucumber-preprocessor/steps"
+import { mockServerClient } from "mockserver-client"
+import { getCurrentFrontendDomain, getCurrentFrontendWebsiteId } from "../../lib/frontend_context"
+
+const Mockclient = mockServerClient("localhost", 1080);
+
+When('I create an account', function() {
```

Comment:
Might be worth extracting this and the one in `devtools_m1/cypress/integration/CustomerCartInteractions/steps.js` into the shared steps.

Reply:
I will do that in a separate PR I think.